repos/system_upgrade/el7toel8/actors/checkmultiplepackageversions/tests/test_multiplepackageversions.py

```diff
@@ -0,0 +1,61 @@
+import mock
+
+from leapp.actors import Actor
+from leapp.models import InstalledRPM, Report, RPM
+
+
+def test_x32_x64(current_actor_context):
+    problem_rpms = [
+       RPM(name='brlapi', version='0.1', release='1.sm01', epoch='1', packager="RH_PACKAGER", arch='i686',
+           pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID 199e2f91fd431d51'),
+       RPM(name='gnome-online-accounts-devel', version='0.1', release='1.sm01', epoch='1',
+           packager="RH_PACKAGER", arch='i686', pgpsig='SOME_OTHER_SIG_X'),
+       RPM(name='geocode-glib-devel', version='0.1', release='1.sm01', epoch='1', packager="RH_PACKAGER",
+           arch='i686', pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID 5326810137017186'),
+       RPM(name='brlapi', version='0.1', release='1.sm01', epoch='1', packager="RH_PACKAGER", arch='x86_64',
+           pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID 199e2f91fd431d51'),
+       RPM(name='gnome-online-accounts-devel', version='0.1', release='1.sm01', epoch='1',
+           packager="RH_PACKAGER", arch='x86_64', pgpsig='SOME_OTHER_SIG_X'),
+       RPM(name='geocode-glib-devel', version='0.1', release='1.sm01', epoch='1', packager="RH_PACKAGER",
+           arch='x86_64',
+           pgpsig='RSA/SHA256, Mon 01 Jan 1970 00:00:00 AM -03, Key ID 5326810137017186')
+       ]
+
+    current_actor_context.feed(InstalledRPM(items=problem_rpms))
+    current_actor_context.run()
+    report = current_actor_context.consume(Report)[0].report
+    assert report['title'] == ('Some packages have both 32bit and 64bit version installed which are known to'
```

Comment:
Not all are incompatible and some are to be fixed so I would suggest changing the sentence to:
`Some packages have both 32bit and 64bit version installed which are known to cause rpm transaction test to fail`

Reply:
Should be fixed